### PR TITLE
Update checkbox.md

### PR DESCRIPTION
**Pull request by Design-Kat**
Astro-581 added documentation for PI updates. Update Checkbox documentation/compliance.

https://rocketcom.atlassian.net/browse/ASTRO-581

### CODE REVIEW

**Comment by Design-Kat** [comment]
@dmcalester @amalyah @Cheshire89 Please review when you get a chance for Astro-581.

File: _content/components/checkbox.md [hunk 1]
```diff
@@ -21,9 +21,12 @@ A Checkbox describes a state or value that can be either “On or Off.” Checkb
 - In a list, each Checkbox is independent of all other Checkboxes.
 - When asking the user to make a mutually exclusive choice, use a Radio Button not a Checkbox.
 - Group Checkboxes whenever possible.
+- When grouped without a parent checkbox, provide a group label.
 
 ## Appearance and Behavior
 
+A basic Checkbox consists of a visual indicator of it's selected state followed by a label. Individual Checkboxes can appear selected (checked), Unselected (no check), and Disabled (no action can be taken by the user). An Indeterminate state (a dash symbol rather than checked) may display when a checkbox is used as a parent of a group of checkboxes where at least one child is selected and at least one is not.
```
Comment (amalyah) [inline _content/components/checkbox.md:28]: "its selected state"

File: _content/components/checkbox.md [hunk 2]
```diff
@@ -37,4 +40,13 @@ Don’t use a Checkbox to initiate an action. Instead, use an [Action Button](/c
 
 ![Don’t: Poorly placed and misaligned Checkboxes make it difficult for users to differentiate one state from another.](/img/components/checkbox-dont-1.png "Don’t: Poorly placed and misaligned Checkboxes make it difficult for users to differentiate one state from another.")
 
+::: two-col
+![Do: Use parent checkboxes, when grouped, to select all, or select none.](/img/components/image.png "Do: Use parent checkboxes, when grouped, to select all, or select none.")
```
Comment (amalyah) [inline _content/components/checkbox.md:44]: "when grouped, to select all or select none."

File: _content/components/checkbox.md [hunk 3]
```diff
@@ -37,4 +40,13 @@ Don’t use a Checkbox to initiate an action. Instead, use an [Action Button](/c
...
+
+![Don’t: Group single checkboxes unless you have a good reason to do so.](/img/components/image.png "Group single checkboxes unless you have a good reason to do so.")
```
Comment (amalyah) [inline _content/components/checkbox.md:46]: Unless you have a good reason to do what?

**Review by amalyah — CHANGES_REQUESTED** [review]
- See comments for content edits
- Need to resolve "This branch is out-of-date with the base branch"
- Can't see preview since checks and deploy failed.

File: _content/components/checkbox.md (hunk 1 above)
Comment (Design-Kat) [reply _content/components/checkbox.md]: Fixed with last commit.

File: _content/components/checkbox.md (hunk 2 above)
Comment (Design-Kat) [reply _content/components/checkbox.md]: Fixed with last commit.

File: _content/components/checkbox.md (hunk 3 above)
Comment (Design-Kat) [reply _content/components/checkbox.md]: Clarified text with last commit.

**Comment by Design-Kat** [comment]
> * See comments for content edits
> * Need to resolve "This branch is out-of-date with the base branch"
> * Can't see preview since checks and deploy failed.

Fixed the updates.
It looks like Duncan had merged the draft branch with this, and so I believe the deploy and checks should pass now?

**Review by amalyah — APPROVED** [review]

File: _content/components/checkbox.md [hunk 4]
```diff
@@ -37,4 +40,13 @@ Don’t use a Checkbox to initiate an action. Instead, use an [Action Button](/c
 
 ![Don’t: Poorly placed and misaligned Checkboxes make it difficult for users to differentiate one state from another.](/img/components/checkbox-dont-1.png "Don’t: Poorly placed and misaligned Checkboxes make it difficult for users to differentiate one state from another.")
 
+::: two-col
+![Do: Use parent checkboxes, when grouped, to select all or select none.](/img/components/image.png "Do: Use parent checkboxes, when grouped, to select all or select none.")
+
+![Don’t: Group a single checkbox under a parent checkbox unless you have a good reason to do so.](/img/components/image.png "Group a single checkbox under a parent checkbox unless you have a good reason to do so.")
+
```
Comment (dmcalester) [inline _content/components/checkbox.md:47]: Please close the two-col with a `:::` here

**Review by dmcalester — CHANGES_REQUESTED** [review]
One minor change to close out the second do/don't pair

**Comment by Design-Kat** [comment]
> One minor change to close out the second do/don't pair

@dmcalester It looks like I missed 2 missing ":::" pairs, I fixed them both right away. Please see the latest commit for the update, thanks!

**Pull request merged**